doc/source/conf.py

```diff
@@ -311,13 +311,17 @@
         "json_url": f"https://{cname}/versions.json",
         "version_match": switcher_version,
     },
-    "cheatsheet": {
+}
+
+BUILD_CHEATSHEET = os.environ.get("BUILD_CHEATSHEET", "true").lower() == "true"
```

Comment:
Following previous discussion, I would go with:

```suggestion
BUILD_CHEATSHEET = os.environ.get("BUILD_CHEATSHEET", "false").lower() == "true"
```

Reply:
So default is false... I dont ming.